packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/Modal/index.js

```diff
@@ -37,7 +38,7 @@ const Modal = ({ handleClose, logId }) => {
     <ModalLayout onClose={handleClose} labelledBy="title">
       <ModalHeader>
         <Breadcrumbs label={formattedDate} id="title">
-          <Crumb>{formattedDate}</Crumb>
+          <Crumb isCurrent>{formattedDate}</Crumb>
         </Breadcrumbs>
```

Comment:
Oh a second instance ... 🤔 Another TODO and maybe create a ticket to replace this with something sematically correct?